.gitignore

```diff
@@ -8,3 +8,7 @@ node_modules
 # Builds
 bin/
 roo-cline-*.vsix
+
+# Local prompts and rules
+prompts
+.clinerules
```

Comment:
Can we remove this too? The idea with `.clinerules` is that we could (and probably should) check in a set of rules that tells Cline how to operate within this repo.

Reply:
but that's always up to each individual's set of rules isn't it?

Reply:
No, the idea is that you can check them into the repository.